honeybee_schema/radiance/material.py

```diff
@@ -0,0 +1,280 @@
+"""Material Schema"""
+from pydantic import Field, constr
+from typing import List, Union
+
+from .._base import IDdBaseModel
+
+
+class Void(IDdBaseModel):
+    """Void modifier"""
+
+    type: constr(regex='^Void$') = 'Void'
+
+
+class ModifierBase(IDdBaseModel):
+    """Base class for Radiance Modifiers"""
+
+    # TODO: We can't define union of schema objects that haven't been
+    # defined yet. We also can't move this down, because ModifierBase
+    # is inherited by all these objects.
+    modifier: Union[
+         Void, Plastic, Glass, BSDF, Glow, Light, Trans
+    ] = Field(
+        # TODO: This needs to be fixed. How can we add add default=Void,
+        # when Void is not serializable?
+        default='void',
+        description='Material modifier (default: Void).'
+        )
+
+    # TODO: We can't define union of schema objects that haven't been
+    # defined yet. We also can't move this down, because ModifierBase
+    # is inherited by all these objects.
+    dependencies: List[
+        Union[Void, Plastic, Glass, BSDF, Glow, Light, Trans]
+    ] = Field(
+        default=[],
```

Comment:
default should be Void.

Reply:
It should be `Void`()` for the `modifier` but is that correct for `dependencies`? You can see here on the honeybee-radiance objects that `dependencies` are currently a list:
https://github.com/ladybug-tools/honeybee-radiance/blob/master/honeybee_radiance/primitive.py#L150

If what you are saying here is correct, then we need to make a change to honeybee-radiance

Reply:
Oh. You are right. My bad.